requirements.txt

```diff
@@ -1,2 +1,3 @@
-pillow==3.0.0
-click==5.1
+pillow>=3.0.0
+click>=5.1
+numpy>=1.10.0
```

Comment:
Unfortunatelly it's a "true and false" statement. writeGif function is converting PIL Image.Image objects to numpy arrays to create subrectangles of each next gif images (basically a compression technique)

It's theoretically possible to pass those independent .

The whole library is a little bit messy - I'll try to make some improvements this weekend.

For now I have removed numpy dependency and "turned off" subRectangles option